.env.sample

```diff
@@ -1,3 +1,6 @@
-L2_GAS_PRICE=4000000 # 0.004 gwei
-L1_GAS_PRICE=40000000000 # 40 gwei
-ETH_PRICE_USD=2500
+L2_GAS_PRICE=60000000 # 0.06 gwei
```

Comment:
these are all numbers I pulled today. Not sure if you prefer another method @jaypaik

Reply:
tbc these are dynamic based on usage so they will change depending on what you pull them, I think both OP and Base contracts are just reporting the current L1 state. We could pull an historical average of these numbers to improve

Reply:
While the base fees change on every L1 block, the scalars (at least on Optimism) seem static. Will look into this more later.